### DISCUSSION TITLE
0.0.2309.101

### DISCUSSION THREAD

**Discussion by prosthetichead** [prosthetichead/GarlicPress#91] [Announcements]
This is a huge update! As always any issues please submit them to the issues tracker!

## What's Changed
* Updated to .Net 7
* Preview widow accessible from Settings by @00anwi 
* Amazing Media Layer Editor  by @00anwi
* Batch downloading from Screen Scraper improving speed by @00anwi
* Region order fixes and ability to change region ordering by @00anwi
* Skin Settings can now upload fonts and boot screens
* Full skin options editing and language file creation and editing
* General fixes and More

## New Contributors
* @00anwi made their first contribution in https://github.com/prosthetichead/GarlicPress/pull/83

**Full Changelog**: https://github.com/prosthetichead/GarlicPress/compare/0.0.2308.141...0.0.2309.101

<hr /><em>This discussion was created from the release <a href='https://github.com/prosthetichead/GarlicPress/releases/tag/0.0.2309.101'>0.0.2309.101</a>.</em>

**Comment by RTEMPS** [comment]
Thank you for implementing this! Whenever i go to save custom box art, it doesnt actually update the art? When i check the device and garlic press it is missing after the update.

**Reply by 00anwi** [reply]
Did you press the Update Art on the home screen with the games you wanted to update the art on?
![image](https://github.com/prosthetichead/GarlicPress/assets/8648447/c70c0434-1f6d-4f64-8bcf-6599bdbebd1f)

And then pressed start? And selected the correct Media Layer Collection (Only needed if you have added multiple)?
And it says Finished Game Art Update?
![image](https://github.com/prosthetichead/GarlicPress/assets/8648447/03ee5d6c-fc72-4bcf-973a-b41a558f30b0)

**Reply by RTEMPS** [reply]
This works for games on screenscraper great but i was trying to use the "open editor" button to add custom artwork. I added my artwork, edited it and clicked saved and it doesnt update. Am i doing it wrong?

**Reply by 00anwi** [reply]
The save is just updating the Media Layer Collection in the background without doing any updates of art in GarlicOS. 
So you first have make your changes in the editor save it and then do the Update Art on the games you want updated.

But I'm adding a button right now to the Editor to directly update the Art in GarlicOS.
This does not exist right now but will hopefully be there in the next version.
![image](https://github.com/prosthetichead/GarlicPress/assets/8648447/78791973-d6ac-497b-b404-07b8313b470b)


I'm working on some more changes after that I will write some documentation for the Media Editor.

**Reply by RTEMPS** [reply]
This is great. Thanks for all your hard work! I still couldnt get custom artwork to save after updating in the media layer and then updating the artwork. It still tries to pull from screenscraper.

**Reply by 00anwi** [reply]
If you have added media from screenscraper it will pull those media assets from screenscraper for each game but local media will not.

So if you want to have custom art work for every game you have to add them as local media the media you click on and add from the ScreenScraper.fr Media tab will update from each game.

Example

Game 1
![image](https://github.com/prosthetichead/GarlicPress/assets/8648447/c106c5ea-41c0-48c6-93c2-8dea44584a08)


Game 2
![image](https://github.com/prosthetichead/GarlicPress/assets/8648447/bc5c5416-ba8a-4523-8bb4-e13fec1fece5)

**Comment by Mustichl** [comment]
This new update is amazing! Having some trouble resizing images for arcade where all the screenshots are different resolutions. Is it possible to resize so all images are 640x480? The width and height settings don't seem to override the resize percentage. Thanks!

**Reply by 00anwi** [reply]
The width and height doesn't do anything right now. It's just displayed right now. I'm not sure what @prosthetichead thought how to use it so I haven't programmed it into the new editor yet.

**Reply by Mustichl** [reply]
Ah makes sense. I think previous versions either automatically resized screenshots or I was able to set them to 640x480. Because I didn't encounter the resolution issue before, too bad as the new gradient feature is exactly what I was doing manually before

**Reply by 00anwi** [reply]
Actually I double checked the code and it is used if resize percentage is 0 and you set width and height. But this will now work yet in the new editor so you may do your changes in the editor and then go into settings and set resize percentage to 0 and the width and height to the size you want. It may be possible to change it in the editor also but I'm not sure how the preview will behave.

**Reply by Mustichl** [reply]
Cool, I'll give it a try. Thanks!
